src/containers/layout/footer/Footer.styles.ts

```diff
@@ -1,19 +1,24 @@
 export const styles = {
   footer: {
     backgroundColor: 'primary.900',
-    color: 'primary.50',
-    marginTop: 'auto'
```

Comment:
why did you removed this style

Reply:
and you don't need to create another pull request to make some changes, just push one more commit in the same pr

Reply:
Thanks

Reply:
> why did you removed this style

my bad(